examples/app-basic/package.json

```diff
@@ -57,10 +57,10 @@
   },
   "dependencies": {
     "@emotion/css": "^11.1.3",
-    "@grafana/data": "10.3.1",
-    "@grafana/runtime": "10.3.1",
-    "@grafana/schema": "10.3.1",
-    "@grafana/ui": "10.3.1",
+    "@grafana/data": "10.2.3",
```

Comment:
This is downgrading dependencies in certain examples. I don't think we want to do that. 🤔